docs/changelog.rst

```diff
@@ -6,6 +6,12 @@ Changelog
 `Unreleased`_
 -------------
 
+Added
+~~~~~
+
+- Fix, check list is non-empty before indexing
```

Comment:
Could you please:

- rephrase it in a way so it reflects changes in user-facing behavior. Was it an error? If yes, then something like "`TypeError`, that happened during processing of empty "examples" list" (and put the proper error type there);
- Replace "Added" section name with "Fixed", since it is a bugfix;
- Remove one extra line after this entry